cognite_toolkit/_cdf_tk/plugins.py

```diff
@@ -0,0 +1,18 @@
+from __future__ import annotations
+
+from enum import Enum
+from functools import lru_cache
+
+from cognite_toolkit._cdf_tk.cdf_toml import CDFToml
+
+
+class Plugins(Enum):
+    NULL = (0,)
+    GRAPHQL = (1,)
```

Comment:
What is the thinking here? Why is the plugin value a tuple?

Reply:
Don't know, think it was added by Ruff.